MAINTAINERS.yml

```diff
@@ -261,6 +261,24 @@ Ambiq Platforms:
   labels:
     - "platform: Ambiq"
 
+AndesTech Platforms:
+  status: maintained
+  maintainers:
+    - jimmyzhe
```

Comment:
you need to follow the process here https://docs.zephyrproject.org/latest/project/project_roles.html and be added to the project first as contributor before this happens

Reply:
Got it, thank you for your guidance.

Hi @kevinwang821020, @wtlee1995, please also nominate yourselves and join the project.